shoes-core/lib/shoes/star.rb

```diff
@@ -39,28 +39,24 @@ def in_bounds?(x, y)
 
     def redraw_left
       return 0 unless element_left
-      if center
-        element_left - width * 0.5 - style[:strokewidth].to_i
-      else
-        super
-      end
+      calculated_left = element_left
+      calculated_left -= width * 0.5 if center
+      calculated_left - strokewidth.ceil * 4
     end
 
     def redraw_top
       return 0 unless element_top
-      if center
-        element_top - width * 0.5 - style[:strokewidth].to_i
-      else
-        super
-      end
+      calculated_top = element_top
+      calculated_top -= width * 0.5 if center
+      calculated_top - strokewidth.ceil * 4
     end
 
     def redraw_width
-      element_width + style[:strokewidth].to_i * 2
+      element_width + strokewidth.ceil * 8
     end
 
     def redraw_height
-      element_height + style[:strokewidth].to_i * 2
+      element_height + strokewidth.ceil * 8
```

Comment:
the 8s and 4s are a bit magical to me here :angel:

Reply:
:+1: